code/modules/mob/living/carbon/alien/adult/adult.dm

```diff
@@ -111,7 +111,7 @@ GLOBAL_LIST_INIT(strippable_alien_humanoid_items, create_strippable_list(list(
 	return ..()
 
 /// Returns FALSE if we're not allowed to eat it, true otherwise
-/mob/living/carbon/alien/adult/proc/can_consume(atom/movable/poor_soul)
+/mob/living/carbon/proc/can_consume(atom/movable/poor_soul)
```

Comment:
And you could have it all, my empire of dirt, 
I will let you down, I will make you hurt.

I wear this crown of thorns, upon my liar's chair,
Full of broken thoughts, I can not repair.

Beneath the stains of time, the feelings disappear,
You are someone else, I am still right here.

What have I become, my sweetest friend,
Everyone I know, goes away in the end.